src/pal/pal_solaris.h

```diff
@@ -19,6 +19,8 @@ namespace snmalloc
      */
     static constexpr uint64_t pal_features = PALPOSIX::pal_features;
 
+    static constexpr size_t page_size =
+      Aal::aal_name == Sparc ? Aal::smallest_page_size : 0x1000;
```

Comment:
As above:

```suggestion
    static constexpr size_t page_size = Aal::smallest_page_size;
```

Reply:
@devnexen  does David's suggestion not work?